MainTel/MainTel-Troubleshooting/MTCS-Troubleshooting/TMA-troubleshooting/TMA-Troubleshooting-OSS-Fails-to-Turn-On-index.rst

```diff
@@ -0,0 +1,181 @@
+
+.. Review the README in this procedure's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Ioana Sotuela*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Karla Aubel*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Templates-Title-of-Troubleshooting-Procedure:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+########################
+TMA OSS Fails to Turn On
+########################
+
+
+.. _Title-of-Troubleshooting-Procedure-Overview:
+
+Overview
+========
+
+.. In one or two sentences, explain when this troubleshooting procedure needs to be used. Describe the symptoms that the user sees to use this procedure. 
+
+This procedure describes how to recover the OSS system when it fails to turn ON. 
+.. _Title-of-Troubleshooting-Procedure-Error-Diagnosis:
+
+Error diagnosis
+===============
+
+.. This section should provide simple overview of known or suspected causes for the error.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Post screenshots of the error state or relevant tracebacks.
+
+- Chiller(s) light in the EUI OSS screen is red. **"OSS"**, **"observation"** and **"cooling"** lights are red too. On the alarm banner on the bottom, you can read **"LOCAL: COOLING SYSTEM"**
+
+.. figure:: ./_static/OSS_general_view.png
+   :name: OSS general view 
+
+
+
+
+
+
+The **Alarm History** screen shows
```

Comment:
Add the description and code of the error shown in Alarm History. It´s just difficult to see in the screenshot.

Reply:
I added a couple of images more in detail. Let me know if like it.